src/Juvix/Compiler/Nockma/Translation/FromSource/Base.hs

```diff
@@ -36,7 +35,7 @@ runParserProgram :: FilePath -> Text -> Either MegaparsecError (N.Program Natura
 runParserProgram = runParserFor program
 
 runParserFor :: Parser a -> FilePath -> Text -> Either MegaparsecError a
-runParserFor p f input = case P.runParser (spaceConsumer >> p <* eof) f input of
+runParserFor p f input' = case P.runParser (spaceConsumer >> p <* eof) f input' of
```

Comment:
Name change

Reply:
This change was moved to https://github.com/anoma/juvix/pull/2582